src/main/java/com/simpligility/maven/plugins/android/phase09package/ApkMojo.java

```diff
@@ -21,6 +21,7 @@
 import com.android.sdklib.build.DuplicateFileException;
 import com.android.sdklib.build.SealedApkException;
 import com.simpligility.maven.plugins.android.AbstractAndroidMojo;
+import com.google.common.io.Files;
```

Comment:
I meant apache commons since I think its already used in the project... just to reduce different tools used. But have a look first .. if its too much effort .. using the google one is fine as well. Btw... is it part of guava?